modules/oe_theme_helper/oe_theme_helper.module

```diff
@@ -92,6 +92,12 @@ function oe_theme_helper_theme($existing, $type, $theme, $path) {
         'name' => '',
       ],
     ],
+    'oe_theme_helper_call_label_status' => [
```

Comment:
Might be we should implement hook_theme_suggestions_HOOK() based on name like `oe_theme_helper_theme_suggestions_oe_theme_helper_call_status`?